report_bundler/test_schema.py

```
@@ -0,0 +1,15 @@
+def test_csv_schema():
+    """
+    Validate that results.csv contains all expected columns:
+    both required and optional metadata fields.
+    """
+    expected_columns = {
+        "image_path", "sampler", "steps", "cfg", "preset", "seed",
+        "width", "height", "grid_label", "notes"
+    }
+
+    with open("results.csv", newline='') as f:
+        reader = csv.DictReader(f)
```

Ruff (0.12.2)

Undefined name `csv`

(F821)